code/style.css

```diff
@@ -1,4 +1,379 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/* MOBILE VIEW <-- 768PX */
+
+body {
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+    font-family: 'Noto Serif JP', serif;
```

Comment:
In this case line 5 doesn't do anything since you've added another font on the next line, so in this case 'Noto Serif' is the default being used.